packages/service-library/src/servicelib/__init__.py

```diff
@@ -2,4 +2,48 @@
 
 """
 
+
+def __monkey_patch_pydantic_url_regex() -> None:
+    # waiting for PR https://github.com/samuelcolvin/pydantic/pull/2512 to be released into
+    # pydantic main codebase
+
+    import pydantic
+
+    if pydantic.VERSION > "1.8.1":
```

Comment:
careful with version strings! I rather compare tuple of ints or using ``packaging.version``
Look
```command
>>> "1.10.2" > "1.8.1"
False
>>> "1.8.2" > "1.8.1"
True


>>> version = lambda s: tuple(map(int,s.split(".")))
>>> version("1.0.2") > version("1.8.1")
False
>>> version("1.10.2") > version("1.8.1")
True

>>> import packaging.version
>>> packaging.version.parse("1.10.2") < packaging.version.parse("1.8.1")
```